File: Dev/ladders.c
#include <stdio.h>

int main() {
    int n;
    printf("input no of terms :: ");
    scanf("%d",&n);
    printf("\n");
    
    for(int i=1; i <= n; i++)
    {
        for(int j=1; j<= i*n; j++)
        {
            if (j > i*n-n)
                printf("*");
            else
                printf(" ");
        }
        printf("\n");
    }
    printf("\n");
    
    return 0;
}

File: Dev/ladders.py
n = int(input("input no of terms :: "))

print()

for i in range(1, n+1):
    for j in range(1, i*n+1):
        if (j > i*n-n):
            print("*",end="")
        else:
            print(" ",end="")
    print()

print()
